Project/core/src/test/java/Model/GameTests.java

```diff
@@ -0,0 +1,121 @@
+package Model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GameTests {
+
+    private Game game;
+    private Feature[] featuresOfInterest;
+    private Player player1;
+    private Player player2;
+    private Board board;
+
+    @BeforeEach
+    void setUp() {
+        // Mocking Player and Board objects
+        featuresOfInterest = new Feature[3];
+        player1 = new Recruiter(0, null, featuresOfInterest);
+        player2 = new RougeAgent(1, null);
+        board = new Board(2,4);
+        List<Player> players = Arrays.asList(player1, player2);
+
+        // Initializing the Game object
+        game = new Game(players, board, player1);
+    }
+
+    @Test
+    void testGameConstructorValidation() {
+        assertThrows(IllegalArgumentException.class, () -> new Game(null, board, player1),
+                "Constructor should throw an exception if players list is null.");
+        assertThrows(IllegalArgumentException.class, () -> new Game(Collections.emptyList(), board, player1),
+                "Constructor should throw an exception if players list is empty.");
+        assertThrows(IllegalArgumentException.class, () -> new Game(Arrays.asList(player1, player2), null, player1),
+                "Constructor should throw an exception if board is null.");
+        assertThrows(IllegalArgumentException.class, () -> new Game(Arrays.asList(player1, player2), board, null),
+                "Constructor should throw an exception if startingPlayer is null.");
+        Player player3 = new RougeAgent(2, "Player 3");
+        assertThrows(IllegalArgumentException.class, () -> new Game(Arrays.asList(player1, player2), board, player3),
+                "Constructor should throw an exception if startingPlayer is not in the players list.");
+    }
+    
+    @Test
+    void testGameInitialization() {
+        assertFalse(game.isGameOver(), "Game should not be over initially.");
+        assertEquals(player1, game.getCurrentPlayer(), "Starting player should be Player 1.");
+        assertEquals(1, game.getCurrentTime(), "Initial game time should be 1.");
+        assertEquals(0, game.getAmountRecruited(), "Amount recruited should start at 0.");
+        assertEquals(0, game.getMindSlipCount(), "Mind slip count should start at 0.");
```

Comment:
Actually the recruiter starts with one mindslip and can gain another from the board. Gaining a new one is only part of the more advanced rules.